tests/mail_test.py

```diff
@@ -7,7 +7,11 @@ def setUp(self):
         password = ''
         assert account != ''
         assert password != ''
-        self.args = (account, password)
+        attachments = []
+        if len(sys.argv) > 3:
+            for i in range(3, len(sys.argv)):
+                attachments.append(sys.argv[i])
+        self.args = (account, password, attachments)
```

Comment:
It should not put `attachments` as an argument.  `SetUp()` wouldn't invoke with contents or attachments in emails.